k3s/k3s-init.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+# Check k3s installation
+k3s_path=$(which k3s)
+if [ -x "$k3s_path" ]; then
+    echo "k3s is already installed"
+    exit 0
+fi
+
+# Install k3s
+$ curl -sfL https://get.k3s.io | K3S_KUBECONFIG_MODE="644" sh -s -
```

Comment:
What is the purpose of the leading `$` here?

Reply:
Also you can avoid the need to change the k3s kubeconfig permissions by using a `sudo cat ${K3S_KUBECONFIG} > /tmp/config.k3s` below to create a temporary copy of the config that the user can access, and then merge that temporary copy with the user's `~/.kube/config` rather than merging the main `${K3S_KUBECONFIG}`.

Reply:
I copied the original command from k3s docs, didn´t notice the `$` 😅.
Also, I was trying to avoid using sudo for this since it would prompt for admin pass when running the script. Since k3s docs provide a way to install it for testing env, I figured it should be ok. Nonetheless, kubectl merge won't alter the original file, so I guess it is fine?

Reply:
Ok, make sense - there are other ways we can approach this